python/taichi/misc/np2ply.py

```diff
@@ -236,3 +246,21 @@ def export_ascii(self, path):
                 for j in range(self.num_face_channels):
                     f.write(str(self.face_data[j][i]) + " ")
                 f.write("\n")
+
+    def export_for_time_series_ascii(self, series_num: int, path):

```

Comment:
Can we make sure path here is a string?

Reply:
Also could you make the name shorter considering this could be the main entrance of this feature? Something like 'export_frame' and 'export_frame_ascii' maybe?

Reply:
> Can we make sure path here is a string?

Right!

Reply:
> Also could you make the name shorter considering this could be the main entrance of this feature? Something like 'export_frame' and 'export_frame_ascii' maybe?

Haha, I apologize for some of my previous experiences from those industrial standards. (like `exportForThisTimeInASCII`).

Reply:
Hahaha, in python everything should be cool